README.md

```diff
@@ -24,9 +26,20 @@ It is meant for discussion, prototype specification and implementation of a prop
 
 * See the [modified spec](https://webassembly.github.io/reference-types/core/) for details.
 
+## Multi-value Proposal for WebAssembly
+
+[![Build Status](https://travis-ci.org/WebAssembly/multi-value.svg?branch=master)](https://travis-ci.org/WebAssembly/multi-value)
+
+This repository is a clone of [github.com/WebAssembly/spec/](https://github.com/WebAssembly/spec/).
+It is meant for discussion, prototype specification and implementation of a proposal to add support for returning multiple values to WebAssembly.
+
+* See the [overview](proposals/multi-value/Overview.md) for a summary of the proposal.
```

Comment:
This link does not point to the correct document. I think we need an absolute address. The same for already committed reference type overview link above.